docker-compose.yml

```diff
@@ -92,3 +92,20 @@ services:
       - EXTRA_CORS_EXPOSE_HEADERS=x-amz-request-id,x-amzn-requestid,x-amz-id-2
       - FORCE_NONINTERACTIVE=true
       - START_WEB=0
+  serverless_localstack:
```

Comment:
Why is a second instance of `localstack` needed?

Reply:
Already discussed in slacked.